spec/models/spree/credit_card_spec.rb

```diff
@@ -57,23 +66,30 @@ module Spree
 
       context "when no card is currently set as the default for a user" do
         context "and I create a new card" do
-          let(:attrs) { { user: user } }
-          let!(:card1) { create(:credit_card, attrs) }
-
           context "without specifying it as the default" do
+            let!(:card1) { create(:credit_card, stored_card_attrs) }
+
             it "sets it as the default anyway" do
               expect(card1.reload.is_default).to be true
```

Comment:
from the PR description I thought we dont want this behaviour anymore. when user creates a new card without specifying it as the default, it will not make it default because it can be a one time card...

Reply:
Sorry for the confusion. There are two ways to create a card:

1. Enter card details during checkout. This creates a one-time card. It should never be the default because it can't be re-used.
2. Add a card in the profile. This always creates a re-usable card. You add it to the profile to store it long term, not just for one checkout. When you add the first card, it should automatically become the default. This saves a couple of clicks.

In this test we create a "stored card" that is re-usable. It should become the default.

Reply:
ah, I get it. thanks.